5.5/run-mysqld.sh

```diff
@@ -94,6 +103,29 @@ if [ "$1" = "mysqld" -a ! -d "$MYSQL_DATADIR/mysql" ]; then
 		EOSQL
 	fi
 	mysqladmin $admin_flags flush-privileges shutdown
+}
+
+function generate_config() {
+	# Time zone.
```

Comment:
There is no such thing. The default is that it's not in the config file at all. I could use another variable with another name, but that seems ugly and unnecessary (one more variable to keep track of..)